Library/Formula/vis.rb

```diff
@@ -0,0 +1,23 @@
+class Vis < Formula
+  desc "Aims to be a modern, legacy free, simple yet efficient vim-like editor."
```

Comment:
You might want to shorten this to `"Modern, legacy free, simple yet efficient vim-like editor"`.